packages/developer-portal/src/components/analytics/costs/index.tsx

```diff
@@ -78,7 +91,30 @@ export const AnalyticsCosts: FC = () => {
 
   return (
     <>
-      <Title>Costs</Title>
+      <FlexContainer isFlexJustifyBetween>
+        <Title>Costs</Title>
+        {isMobile && (
+          <ButtonGroup alignment="right">
+            <Button intent="low" onClick={openModal}>
+              Controls
+            </Button>
+          </ButtonGroup>
+        )}
+      </FlexContainer>
+      {isMobile && (
+        <Modal title="Controls">
+          <Controls />
+          <ButtonGroup alignment="center">
+            <Button fixedWidth intent="secondary" onClick={closeModal}>
+              Close
+            </Button>
+          </ButtonGroup>
+        </Modal>
+      )}
+      <Subtitle>
+        Total for period: £
+        {aggregatedBilling.totalCost ? aggregatedBilling.totalCost.toFixed(2).padStart(2, '0') : '£0'}
```

Comment:
pound sign already exists? So result for no totalCost would be `£ £0`

Reply:
Yep, good spot

Reply:
Yep, good spot